EventsExpress/Controllers/UsersController.cs

```diff
@@ -161,15 +161,16 @@ public async Task<IActionResult> Block(Guid userId)
         /// <summary>
         /// This method is to edit username.
         /// </summary>
-        /// <param name="userInfo">Required.</param>
+        /// <param name="userName">Required.</param>
         /// <response code="200">Edit is succesful.</response>
         /// <response code="400">Edit process failed.</response>
         [HttpPost("[action]")]
-        public async Task<IActionResult> EditUsername(UserInfo userInfo)
+        public async Task<IActionResult> EditUsername(EditUserNameDto userName)
         {
-            var validator = new UserInfoNameValidator();
+            var validator = new EditUserNameDtoValidator();
```

Comment:
You don't need to inline validator calls. These will be done by registering fluent validation from `Startup.cs`.